code/server/test/api_endpoint_tests.js

```diff
@@ -1,26 +1,29 @@
 var app = require('../main.js');
 var request = require('supertest');
 var assert = require('assertthat'); // View README for documentation https://github.com/thenativeweb/assertthat
-
+var api = require('../routes/api.js');
+var stubUser = require('../stub_models/StubUser.js');
+var stubLikes = require('../stub_models/StubUserMatches.js');
 //////////////////////////////////////////
 // ALL API TESTS SHOULD GO IN THIS FILE // 
 //////////////////////////////////////////
 
 describe('api', function() {
+          api.injectUser(stubUser);
+          api.injectLikes(stubLikes);
```

Comment:
main.js doesn't give any access to main's api import.  Could refactor the test to not import main, and just use the api import, I guess.